gcloud/dns/changes.py

```diff
@@ -221,7 +221,8 @@ def create(self, client=None):
         self._set_properties(api_response)
 
     def exists(self, client=None):
-        """API call:  test for the existence of the change set via a GET request
+        """API call:  test for the existence of the change set via a GET
+        request.
```

Comment:
It's up to you, but I really want to call out the fact that the 'single line description should be a single line' is broken *everywhere* in the Python community. Requests, urllib3, github3.py, etc. Sphinx has explicit support for the 'short description' being multiple lines.